common/app/model/RelatedContent.scala

```diff
@@ -15,28 +15,11 @@ case class RelatedContentItem (
   faciaContent: PressedContent
 )
 
-case class RelatedContent ( items: Seq[RelatedContentItem], storyPackagesCount: Int = 0) {
```

Comment:
`storyPackagesCount` is not useful anymore. It was only used here https://github.com/guardian/frontend/pull/12938/files#diff-79e70845e1ad76d8d445887a5498a2fdL15